wp/headless-wp/includes/classes/Integrations/Gutenberg.php

```diff
@@ -19,6 +19,75 @@ class Gutenberg {
 	 */
 	public function register() {
 		add_filter( 'render_block', [ $this, 'render_block' ], 10, 3 );
+		add_filter( 'render_block_core/image', [ $this, 'ensure_image_has_dimensions' ], 9999, 2 );
+	}
+
+	/**
+	 * Get the image ID by URL
+	 *
+	 * @param string $url The image URL
+	 *
+	 * @return int
+	 */
+	protected function get_image_by_url( $url ) {
+		if ( function_exists( '\wpcom_vip_url_to_postid' ) ) {
```

Comment:
@nicholasio I had to change it to wpcom_vip_attachment_url_to_postid instead to make it work